code/modules/mob/living/status_procs.dm

```diff
@@ -41,9 +41,9 @@
 
 /mob/living/proc/Stun(amount, ignore_canstun = FALSE) //Can't go below remaining duration
 	if(SEND_SIGNAL(src, COMSIG_LIVING_STATUS_STUN, amount, ignore_canstun) & COMPONENT_NO_STUN)
-		return
+		return FALSE
 	if(check_stun_immunity(CANSTUN, ignore_canstun))
-		return
+		return FALSE
```

Comment:
These procs don't need to return `FALSE`
They return `null | status_effect`, which is ultimately the same thing as returning `FALSE | status_effect` as far as checking success or failure goes
Generally the former is preferred because if you have a proc check `Stun()?.duration` it will runtime with your change but work without your change